app/javascript/components/widgets/fires/fires-within/index.js

```diff
@@ -80,7 +80,8 @@ export default {
     dataset: 'viirs',
     confidence: 'h',
     landCategory: '',
-    forestType: 'ifl'
+    forestType: '',
+    showSettingsBtn: true
```

Comment:
This isnt a setting as it doesnt affect the mapping of the data or the settings menu values. This should be present at the top level of the widget